src/js_realm.hpp

```diff
@@ -390,6 +393,28 @@ void RealmClass<T>::constructor(ContextType ctx, ObjectType this_object, size_t
                 config.schema_version = 0;
             }
 
+            static const String compact_on_launch_string = "shouldCompactOnLaunch";
+            ValueType compact_value = Object::get_property(ctx, object, compact_on_launch_string);
```

Comment:
I am not sure if this will work automatically with the chrome debugger proxies, try running the tests with 
```
scripts/test.sh react-tests Debug
```